cfme/tests/containers/test_provider_configuration_menu.py

```diff
@@ -22,3 +22,14 @@ def test_edit_selected_containers_provider(provider):
     view = navigate_to(provider, 'Edit')
     assert view.is_displayed
     view.cancel.click()
+
+
+def test_ocp_operator_out_of_the_box(appliance):
+
```

Comment:
**REQUIRED**
Please remove that blank line.

Reply:
@quarckster  Ipreffet to add a doc string instead of on removing this blank, that would keep the code readable.

Reply:
So please add the docstring. Don't keep lines blank.

Reply:
@quarckster Done